packages/webapp/src/routes/land/any-other-landowners.js

```diff
@@ -0,0 +1,49 @@
+import constants from '../../utils/constants.js'
+import { checked, getLegalAgreementDocumentType } from '../../utils/helpers.js'
+
+const href = '#anyOtherLO-yes'
+const handlers = {
+  get: async (request, h) => h.view(constants.views.ANY_OTHER_LANDOWNERS, {
+    ...getContext(request),
+    checked
+  }),
+  post: async (request, h) => {
+    const anyOtherLOValue = request.payload.anyOtherLOValue
+    request.yar.set(constants.redisKeys.ANY_OTHER_LANDOWNERS_CHECKED, anyOtherLOValue)
+    if (anyOtherLOValue === 'Yes') {
+      return h.redirect(constants.routes.LANDOWNER_CONSERVATION_COVENANT_INDIVIDUAL_ORGANISATION)
+    } else if (anyOtherLOValue === 'No') {
+      return h.redirect(constants.routes.HABITAT_PLAN_LEGAL_AGREEMENT)
+    } else {
+      return h.view(constants.views.ANY_OTHER_LANDOWNERS, {
+        ...getContext(request),
+        checked,
+        err: [
+          {
+            text: 'Select yes if there are any other landowners or leaseholders',
+            href
+          }
+        ]
+      })
+    }
+  }
+}
+
+const getName = account => `${account.idTokenClaims.firstName} ${account.idTokenClaims.lastName}`
```

Comment:
This assumes the logged in user is the landowner or leaseholder.

- If the user is logged in representing an organisation I believe the landowner or leaseholder should be the represented organisation.
- Regardless of how the user is logged in,  If the user is acting as agent I believe the landowner or leaseholder should be the client name  (individual or organisation)

Additionally, if I am correct and the user completes the legal agreement details before the applicant information there is no way of knowing who the landowner or leaseholder is until the applicant details are completed.

UCD input is required for resolution.